.travis/install.sh

```diff
@@ -16,3 +16,4 @@ fi
 command -v python3
 python3 --version
 python3 -c "import platform; print('macOS version:', platform.mac_ver()[0])"
+pip3 install codecov
```

Comment:
This should also move to the `tests.sh` script, where the other pip installs are being done.

(I'm not sure our current setup is good, but I don't think we can pip install codecov before our venv is activated.)

Reply:
But you also can’t run the test script without codecov being installed: `coverage run "./.travis/tests.sh"`

Reply:
Sorry, I didn't read your previous comment. Both ways should work.

Reply:
No, we shouldn't run coverage on pip install etc... The `pip install codecov` _and_ the  `coverage run ...` call should move to tests.sh.

(Also: `coverage` is a Python script runner. It doesn't work with shell scripts.)

Reply:
Okay, I moved it like you said 🤞